docs/guides/search-path.md

```diff
@@ -106,7 +111,7 @@ Steampipe will add the aggregator before the other aws connections because `aws`
 +-------------------------------------+
 ```
 
-If you prefer, you can explicitly set the search path in the [database options ](https://steampipe.io/docs/reference/config-files/database) in your `~/.steampipe/config/default.spc` file.  Note that this is somewhat brittle though - every time you install or uninstall a plugin, add or remove a connection, etc you will need to update the file with the new search_path.
+If you prefer, you can explicitly set the search path in the [database options ](https://steampipe.io/docs/reference/config-files/database) in your `~/.steampipe/config/default.spc` file.  Note that this is somewhat brittle. Every time you install or uninstall a plugin, add or remove a connection, etc you will need to update the file with the new search_path.
```

Comment:
semi-colon or dash is better than a period here, as the second "sentence" explains the first

Reply:
How about: "Note that this is somewhat brittle because every time you install or uninstall a plugin, or add or remove a connection, you will need to update the file with the new search_path."